app/views/orders/_index_table.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="table-responsive">
+  <table class="table table-striped data-table">
+    <thead>
+      <tr>
+        <th class="sort-asc">Order Number</th>
+        <th>County</th>
+        <th>Organization Name</th>
+        <th>Date Received</th>
+        <th>Status</th>
+        <th>Tracking Number</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% if @orders.present? %>
```

Comment:
Could you pass `@orders` in as a local for this partial? I tend to prefer partials to receive the variables they use for rendering rather than expect the controller to set up special variables. This makes it more clear what variables the partial depends on at the places it is used, and tends to be more flexible in how it is used.

Also, I think `index` in the name is unnecessary... this seems like it would be fine as just the `"orders/table"` partial.

Reply:
👍